includes/class-ajax.php

```diff
@@ -109,9 +109,10 @@ public function save_form() {
         $integrations  = array();
 
         if ( isset( $post_data['settings'] ) ) {
-            $settings = (array) json_decode( $post_data['settings'] );
-            $settings['message'] = sanitize_text_field( $settings['message'] );
-            error_log(print_r($settings, true));
+            $settings                  = (array) json_decode( $post_data['settings'] );

```

Comment:
https://www.php.net/manual/en/function.json-decode.php

instead of using (array) to coerce an array, you should use json_encode( $post_data['settings'], true )
This uses the json_encode method to decode it into an array directly